spec/rails_helper.rb

```diff
@@ -30,6 +30,8 @@
   exit 1
 end
 
+puts "ℹ️ If you change CSS, JS, or Assets - don't forget to run `RAILS_ENV=test rails webpacker:compile` before your test runs"
```

Comment:
Since you defined a compile_assets task in the rake file, we could just run `be rake compile_assets` 
```suggestion
puts "ℹ️ If you change CSS, JS, or Assets - don't forget to run `rake compile_assets` before your test runs"
```